### PR TITLE
Docs(hash_functions): fix `xxHash32`, `xxHash64`

### PR DESCRIPTION
**Pull request by godfreyd**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Category (leave one):

- Documentation

Fix `xxHash32`, `xxHash64` functions descriptions.

### CODE REVIEW

File: docs/en/query_language/functions/hash_functions.md [hunk 1]
```diff
@@ -305,10 +305,42 @@ SELECT murmurHash3_128('example_string') AS MurmurHash3, toTypeName(MurmurHash3)
...
-For more information, see the link: [xxHash](http://cyan4973.github.io/xxHash/)
+Calculates `xxHash` from a string. It is proposed in two flavors, 32 and 64 bits.
 
-[Original article](https://clickhouse.yandex/docs/en/query_language/functions/hash_functions/) <!--hide-->
+```sql
+SELECT xxHash32('srting');
+
+OR
+
+SELECT  xxHash64('srting');
```
Comment (alexey-milovidov) [inline docs/en/query_language/functions/hash_functions.md]: Double whitespace.

File: docs/en/query_language/functions/hash_functions.md [hunk 2]
```diff
@@ -305,10 +305,42 @@ SELECT murmurHash3_128('example_string') AS MurmurHash3, toTypeName(MurmurHash3)
...
+## xxHash32, xxHash64 {#hash_functions-xxhash32}
 
-Calculates xxHash from a string.
-Accepts a String-type argument. Returns UInt64 Or UInt32.
-For more information, see the link: [xxHash](http://cyan4973.github.io/xxHash/)
+Calculates `xxHash` from a string. It is proposed in two flavors, 32 and 64 bits.
 
-[Original article](https://clickhouse.yandex/docs/en/query_language/functions/hash_functions/) <!--hide-->
+```sql
+SELECT xxHash32('srting');
```
Comment (alexey-milovidov) [inline docs/en/query_language/functions/hash_functions.md]: Why do we need `srting` in this example?

File: docs/en/query_language/functions/hash_functions.md [hunk 3]
```diff
@@ -305,10 +305,42 @@ SELECT murmurHash3_128('example_string') AS MurmurHash3, toTypeName(MurmurHash3)
 └──────────────────┴─────────────────┘
 ```
 
-## xxHash32, xxHash64
+## xxHash32, xxHash64 {#hash_functions-xxhash32}
 
-Calculates xxHash from a string.
-Accepts a String-type argument. Returns UInt64 Or UInt32.
-For more information, see the link: [xxHash](http://cyan4973.github.io/xxHash/)
+Calculates `xxHash` from a string. It is proposed in two flavors, 32 and 64 bits.
```
Comment (alexey-milovidov) [inline docs/en/query_language/functions/hash_functions.md]: What version of xxHash is in use?

File: docs/ru/query_language/functions/hash_functions.md [hunk 4]
```diff
@@ -310,10 +310,42 @@ SELECT murmurHash3_128('example_string') AS MurmurHash3, toTypeName(MurmurHash3)
...
+## xxHash32, xxHash64 {#hash_functions-xxhash32-xxhash64}
 
-Вычисляет xxHash от строки.
-Принимает аргумент типа String. Возвращает значение типа Uint64 или Uint32.
-Дополнительные сведения см. по ссылке: [xxHash](http://cyan4973.github.io/xxHash/)
+Вычисляет `xxHash` от строки. Предлагается в двух вариантах: 32 и 64 бита.
 
-[Оригинальная статья](https://clickhouse.yandex/docs/ru/query_language/functions/hash_functions/) <!--hide-->
+```sql
+SELECT xxHash32('srting');
```
Comment (alexey-milovidov) [inline docs/ru/query_language/functions/hash_functions.md]: .

File: docs/ru/query_language/functions/hash_functions.md [hunk 5]
```diff
@@ -310,10 +310,42 @@ SELECT murmurHash3_128('example_string') AS MurmurHash3, toTypeName(MurmurHash3)
...
+
+OR
+
+SELECT  xxHash64('srting');
```
Comment (alexey-milovidov) [inline docs/ru/query_language/functions/hash_functions.md]: .

File: docs/en/query_language/functions/hash_functions.md (hunk 1 above)
Comment (godfreyd) [reply docs/en/query_language/functions/hash_functions.md]: Done

File: docs/en/query_language/functions/hash_functions.md (hunk 2 above)
Comment (godfreyd) [reply docs/en/query_language/functions/hash_functions.md]: Done

File: docs/en/query_language/functions/hash_functions.md (hunk 3 above)
Comment (filimonov) [reply docs/en/query_language/functions/hash_functions.md]: By default - it's xxHash v0.6.5 from lz4, unless some custom build is used. 
https://github.com/lz4/lz4/blob/7a4e3b1fac5cd9d4ec7c8d0091329ba107ec2131/lib/xxhash.h#L152
Afaik different xxhash versions are compatible.

By the way, @alexey-milovidov : I know you like different hashing algorithms :) xxHash author had presented a new algorithm that year (still considered experimental) - see https://fastcompression.blogspot.com/2019/03/presenting-xxh3.html

File: docs/en/query_language/functions/hash_functions.md [hunk 6]
```diff
@@ -310,11 +310,11 @@ SELECT murmurHash3_128('example_string') AS MurmurHash3, toTypeName(MurmurHash3)
 Calculates `xxHash` from a string. It is proposed in two flavors, 32 and 64 bits.
 
 ```sql
-SELECT xxHash32('srting');
+SELECT xxHash32();
```
Comment (filimonov) [inline docs/en/query_language/functions/hash_functions.md]: What a hell is that? :) What a hash does it return when called w/o params? And why it differs from empty string? :) 

```
SELECT     xxHash32(),     xxHash32(''),     xxHash64(),     xxHash64('');

Row 1:
──────
xxHash32():   4263699484
xxHash32(''): 46947589
xxHash64():   16324913028386710556
xxHash64(''): 17241709254077376921
```

Comment (godfreyd) [reply docs/en/query_language/functions/hash_functions.md]: Done

File: docs/en/query_language/functions/hash_functions.md (hunk 3 above)
Comment (godfreyd) [reply docs/en/query_language/functions/hash_functions.md]: Thanks!

File: docs/en/query_language/functions/hash_functions.md (hunk 6 above)
Comment (godfreyd) [reply docs/en/query_language/functions/hash_functions.md]: Replaced `xxHash32()`  → `xxHash32('')`.

**Pull request merged**